server/api.go

```diff
@@ -462,6 +476,13 @@ func (a *API) oauthRedirectHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	err = a.p.store.DeleteUserInvite(mmUserID)
```

Comment:
This got me thinking: what happens if the user disconnects? Should we suppress inviting them again?

Reply:
Once they've connected for the first time, they're added to the whitelist and not invited again.